RELEASE.md

```diff
@@ -0,0 +1,49 @@
+# Releasing
+
+This repository uses a branch-based strategy to manage releases.
+
+> **Note**
+> This strategy has been introduced later on in this repository's lifetime, so not all releases have an associated branch.
+
+## Naming scheme
+
+A release branch is identified as follows:
+
+`release-{VERSION}{CANDIDATE}`
+
+where `VERSION` is a semantic version in the form of `vx.y.z`, and `CANDIDATE` is the optional `-rc` tag, indicating that it is a release candidate.
+
+- `release-v3.2.1-rc` is the candidate to become `release-v3.2.1`.
+
+## Creating a release
+
+1. Create a PR to the branch of `release-vx.y.z-rc` from `main`, where the version is the next semantic version of the repository depending on changes.
+
+```
+release-vx.y.z-rc <- main
+```
+
+2. Automation will test the release candidate and run integration tests for this release specifically.
+
+    - If fixes need to be applied to this release branch; PR the fixes to main, and cherry-pick the commits into the release candidate.
```

Comment:
If foo bar, please read `## Backporting fixes` ?

Reply:
I'm just putting this here twice, as this is often overlooked.